### PR TITLE
2409: Improve poi category filter layout

### PR DESCRIPTION
**Pull request by f1sh1918**
### Short description

Depending on the amount of categories the filter layout of the toggle buttons look weird/displaced

### Proposed changes

The amount of icons per row is now depending on the width of the panel and implemented with a grid layout
Items of the next should should align left.

### Side effects

Hopefully none

### Resolved issues

#2409 

Desktop Panel
image

mobile
image

iPadMini (768px width before it breaks to desktop)
image

### CODE REVIEW

File: web/src/components/ModalContent.tsx [hunk 1]
```diff
@@ -19,6 +19,7 @@ const Header = styled.div<{ small: boolean }>`
   justify-content: space-between;
   font-size: ${props => props.theme.fonts.subTitleFontSize};
   font-weight: bold;
+  align-items: center;
```
Comment (f1sh1918) [inline web/src/components/ModalContent.tsx:22]: modal content header wasn't aligned properly (icons + text)

File: web/src/components/base/ToggleButton.tsx [hunk 2]
```diff
@@ -3,13 +3,14 @@ import styled from 'styled-components'
 
 import StyledSmallViewTip from '../StyledSmallViewTip'
 
+export const toggleButtonWidth = 100
```
Comment (steffenkleinle) [inline web/src/components/base/ToggleButton.tsx:6]: When does it go to dimensions and when not?

File: web/src/components/base/ToggleButton.tsx [hunk 3]
```diff
@@ -3,13 +3,14 @@ import styled from 'styled-components'
...
 const StyledButton = styled.button<{ $active: boolean | null }>`
   border: none;
   box-shadow:
     0 1px 2px rgba(0, 0, 0, 0.25),
     0 1px 4px 1px rgba(0, 0, 0, 0.15);
   border-radius: 18px;
-  width: 100px;
+  width: ${toggleButtonWidth}px;
   height: 80px;
```
Comment (steffenkleinle) [inline web/src/components/base/ToggleButton.tsx:14]: And why not the width?

**Review by steffenkleinle — APPROVED** [review]
Tested on firefox web and mobile, looks way better :+1: :magic_wand:

File: web/src/components/PoiFilters.tsx [hunk 4]
```diff
@@ -104,7 +108,7 @@ const PoiFilters = ({
             <SubTitle>{t('poiCategories')}</SubTitle>
             <SortingHint>{t('alphabetLetters')}</SortingHint>
           </Row>
-          <TileRow>
+          <TileRow itemCount={Math.floor(panelWidth / (toggleButtonWidth + tileColumnGap))}>
```
Comment (LeandraH) [inline web/src/components/PoiFilters.tsx:111]: Nice solution!

**Review by LeandraH — APPROVED** [review]
Nice, tested in Firefox

**Pull request merged**